src/mono/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -177,4 +177,57 @@
       <attribute internal="RemoveAttributeInstances" />
     </type>
   </assembly>
+
+  <!-- Metadata attributes present in most core SDKs assemblies -->
```

Comment:
What about `IsTrimmable`? Should we be removing those as well?

Reply:
I'm open to suggestions. I was thinking about removing it but don't have a strong preference on this one.

Reply:
I think if we are removing unnecessary attributes, I kind of don't see a reason to keep it around anymore. Are we expecting to trim an assembly more than once?

Reply:
No, we don't. I'll include it then too.